installers/server/release/server.sh

```diff
@@ -112,21 +112,53 @@ fi
 if [ ! -z $SERVER_LISTEN_HOST ]; then
     GO_SERVER_SYSTEM_PROPERTIES="$GO_SERVER_SYSTEM_PROPERTIES -Dcruise.listen.host=$SERVER_LISTEN_HOST"
 fi
-SERVER_STARTUP_ARGS+=("-server $YOURKIT")
-SERVER_STARTUP_ARGS+=("-Xms$SERVER_MEM -Xmx$SERVER_MAX_MEM -XX:PermSize=$SERVER_MIN_PERM_GEN -XX:MaxPermSize=$SERVER_MAX_PERM_GEN")
-SERVER_STARTUP_ARGS+=("$JVM_DEBUG $GC_LOG $GO_SERVER_SYSTEM_PROPERTIES")
-SERVER_STARTUP_ARGS+=("-Duser.language=en -Djruby.rack.request.size.threshold.bytes=30000000")
-SERVER_STARTUP_ARGS+=("-Duser.country=US -Dcruise.config.dir=$GO_CONFIG_DIR -Dcruise.config.file=$GO_CONFIG_DIR/cruise-config.xml")
-SERVER_STARTUP_ARGS+=("-Dcruise.server.port=$GO_SERVER_PORT -Dcruise.server.ssl.port=$GO_SERVER_SSL_PORT")
+SERVER_STARTUP_ARGS+=("-server")
+if [ "$YOURKIT" != "" ]; then
+   GO_SERVER_SYSTEM_PROPERTIES="$GO_SERVER_SYSTEM_PROPERTIES $YOURKIT"
+fi
+SERVER_STARTUP_ARGS+=("-Xms$SERVER_MEM")
+SERVER_STARTUP_ARGS+=("-Xmx$SERVER_MAX_MEM")
+SERVER_STARTUP_ARGS+=("-XX:PermSize=$SERVER_MIN_PERM_GEN")
+SERVER_STARTUP_ARGS+=("-XX:MaxPermSize=$SERVER_MAX_PERM_GEN")
+if [ "$JVM_DEBUG" != "" ]; then
+   GO_SERVER_SYSTEM_PROPERTIES="$GO_SERVER_SYSTEM_PROPERTIES $JVM_DEBUG"
+fi
+if [ "$GC_LOG" != "" ]; then
+   GO_SERVER_SYSTEM_PROPERTIES="$GO_SERVER_SYSTEM_PROPERTIES $GC_LOG"
+fi
+SERVER_STARTUP_ARGS+=("-Duser.language=en")
+SERVER_STARTUP_ARGS+=("-Djruby.rack.request.size.threshold.bytes=30000000")
+SERVER_STARTUP_ARGS+=("-Duser.country=US")
+SERVER_STARTUP_ARGS+=("-Dcruise.config.dir=$GO_CONFIG_DIR")
+SERVER_STARTUP_ARGS+=("-Dcruise.config.file=$GO_CONFIG_DIR/cruise-config.xml")
+SERVER_STARTUP_ARGS+=("-Dcruise.server.port=$GO_SERVER_PORT")
+SERVER_STARTUP_ARGS+=("-Dcruise.server.ssl.port=$GO_SERVER_SSL_PORT")
+
+if [ "$GO_CONFIG_REPO_GC_CRON" != "" ]; then
```

Comment:
can we not do another environment variable for this?

I'm thinking (at the back of my head) about a better way to configure stuff. In the meanwhile, it'd be better that we don't add one more thing here.